templates/pipelines/common-app-ci.yaml

```diff
@@ -493,8 +493,15 @@ stages:
                             FailOnStandardError: true
                             azurePowerShellVersion: 'LatestVersion'
                             pwsh: true
-                            workingDirectory: '$(Pipeline.Workspace)/s'     
-          
+                            workingDirectory: '$(Pipeline.Workspace)/s'        
+
+                      - ${{ if eq(coalesce(deploymentEnv.teamResourcesPermissions.grantTeamAccessGroupPermissionToTeamResourceGroup, 'false'), true) }}:
```

Comment:
I'm not sure if this is the right Pipeline to do this. Mainly focus on app builds and deployments. We have other infrastructure and RBAC pipelines etc. Can we review this @kbabigumira @ajaysoni-cap @ppotturi

Reply:
@DaRoszko : 
This is only for team resourcegroup role assignment and is dependent on the existence of Resource group and Team. 

While Flux is the preferred tool for this task, we are currently unable to utilize it due to optimization issues. Creating a separate pipeline to execute this script is not feasible as it would still require being called from the CI pipeline or manual intervention. 

I discussed this with the team, and we believe we can maintain this as a temporary solution. 
Also please note that this task is not dependent on callback api. https://dev.azure.com/defragovuk/DEFRA-FFC/_build/results?buildId=609882&view=logs&j=c4455180-6351-58ac-f388-f78c673f88c7&t=26ded4e9-3b82-5689-9b2c-95cded21359c